src/AudioToolbox/AudioType.cs

```diff
@@ -705,6 +717,17 @@ internal unsafe AudioChannelLayout (IntPtr h)
 			}
 		}
 
+		public AudioChannelLayoutNative NativeStruct {
+			get {
+				return new AudioChannelLayoutNative {
```

Comment:
From the old PR:

rolfbjarne added a note on Feb 15
Does this have to be public? Seems like we can keep this property (and AudioChannelLayoutNative) internal.